salt/metalk8s/salt/master/files/salt-master-manifest.yaml.j2

```diff
@@ -82,7 +81,8 @@ spec:
     - name: salt-api
       image: {{ image }}
       command: ['salt-api']
-      args: ['--log-level=info']
+      args:
+        - --log-level={{ 'debug' if debug else 'info' }}
```

Comment:
Salt-api do not use the value from master config ?

Reply:
It's a good question! Not clearly stated in documentation, and I didn't test...

Reply:
It does :) Thanks for the tip!